helm-chart-sources/edge/templates/daemonset.yaml

```diff
@@ -79,7 +79,7 @@ spec:
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}{{ if .Values.image.wolfiImage }}-wolfi{{ end }}"
```

Comment:
Can we extend UT in `helm-chart-sources/edge/tests/daemonset_test.yaml` to cover new logic:
Even something simple like that:
```
  - it: Appends -wolfi suffix when wolfiImage is true
    set:
      image:
        repository: foo/bar
        tag: 42.0.0
        wolfiImage: true
    asserts:
      - equal:
          path: spec.template.spec.containers[0].image
          value: foo/bar:42.0.0-wolfi

  - it: Does not append -wolfi suffix when wolfiImage is false
    set:
      image:
        repository: foo/bar
        tag: 42.0.0
        wolfiImage: false
    asserts:
      - equal:
          path: spec.template.spec.containers[0].image
          value: foo/bar:42.0.0
```